src/Cake.Npm/NpmSettingsExtensions.cs

```diff
@@ -48,5 +48,51 @@ public static NpmSettings FromPath(this NpmSettings settings, DirectoryPath path
 
             return settings;
         }
+
+        /// <summary>
+        /// Sets the StandardError-Action
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="standardErrorAction">The StandardError-Action.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="Core.Tooling.ToolSettings.WorkingDirectory"/> set to <paramref name="path"/>.</returns>
```

Comment:
Documents the wrong property which is set

Reply:
corrected.